example_rails7/app/javascript/application.js

```diff
@@ -0,0 +1,3 @@
+// Entry point for the build script in your package.json
+import "@hotwired/turbo-rails"
+import "./controllers"
```

Comment:
This won't actually compile with our custom Webpack config because it requires file extensions. This file should be completely recreated by the webpack generator

```suggestion
import "./controllers/index.js"
```